testsuite/MDAnalysisTests/utils/test_streamio.py

```diff
@@ -259,8 +255,10 @@ class _StreamData(object):
     }
 
     def __init__(self):
-        self.buffers = {name: "".join(open(fn).readlines())
-                        for name, fn in self.filenames.items()}
+        self.buffers = {}
```

Comment:
also did a little bit of cleaning to remove some of the excess warnings being raised - this originally left a bunch of open file handles so I changed it to use context managers.